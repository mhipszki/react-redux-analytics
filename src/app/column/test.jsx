import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import Column from './';

test('styles a div element as a flexbox column', () => {
  const component = shallow(<Column />);
  expect(component).toMatchSnapshot();
});
