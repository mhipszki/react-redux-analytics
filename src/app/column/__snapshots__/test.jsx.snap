// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styles a div element as a flexbox column 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
/>
`;
